resources/css/bem/wiki-search.less

```diff
@@ -12,15 +12,19 @@
   position: relative;
 
   &__bar {
-    .default-border-radius();
     display: flex;
     flex-direction: row;
 
     padding: 13px @_gutter;
     font-size: @font-size--wiki-search;
     background-color: @osu-colour-b6;
 
-    border: @_border-size solid @osu-colour-orange-1;
+    border: 1px solid @osu-colour-b6;
+    
+    &:focus-within {
+      box-shadow: 0 0 10px @osu-colour-h2;
```

Comment:
matching the colour with beatmap search would be preferred but I'll probably refactor the search box later anyway so it's probably fine for now (if not a bit weird when it changes again later)